File: fix_footer.js
import fs from 'fs';
let content = fs.readFileSync('src/components/operations/OperationsLeads.tsx', 'utf-8');

content = content.replace(
  '<div className="p-4 border-t border-zinc-800 flex justify-end gap-3 bg-zinc-950/40">',
  '<div className="p-4 border-t border-zinc-800 flex flex-col sm:flex-row justify-end gap-3 bg-zinc-950/40">'
);

content = content.replace(
  '<button\n                  type="button"\n                  onClick={() => setAssigningOrderId(null)}\n                  className="px-4 py-2 text-xs font-mono font-bold text-zinc-400 hover:text-white transition-colors cursor-pointer"\n                >',
  '<button\n                  type="button"\n                  onClick={() => setAssigningOrderId(null)}\n                  className="px-4 py-2 text-xs font-mono font-bold text-zinc-400 hover:text-white transition-colors cursor-pointer w-full sm:w-auto"\n                >'
);

content = content.replace(
  '<button\n                  type="submit"\n                  disabled={isSaving}\n                  className="px-6 py-2 bg-amber-500 hover:bg-amber-600 text-black text-xs font-mono font-bold uppercase rounded-lg transition-colors cursor-pointer disabled:opacity-50"\n                >',
  '<button\n                  type="submit"\n                  disabled={isSaving}\n                  className="px-6 py-2 bg-amber-500 hover:bg-amber-600 text-black text-xs font-mono font-bold uppercase rounded-lg transition-colors cursor-pointer disabled:opacity-50 w-full sm:w-auto"\n                >'
);

fs.writeFileSync('src/components/operations/OperationsLeads.tsx', content, 'utf-8');
console.log("Fixed modal footer");

File: fix_footer.py
import re

with open("src/components/ProductionModule.tsx", "r") as f:
    content = f.read()

target_start = """            {/* Footer */}"""
target_end = """              </button>
            </div>
          </div>
        </div>
      )}"""

start_idx = content.find(target_start)
end_idx = content.find(target_end, start_idx) + len("""              </button>
            </div>
          </div>
        </div>
      )}""")

if start_idx == -1 or end_idx == -1:
    print("Could not find section.")
    exit(1)

new_content = """            {/* Footer */}
            <div className="p-4 border-t border-zinc-900 bg-[#0c0d10] flex justify-end gap-3">
              <button
                type="button"
                onClick={() => {
                  setEditorWhatsappModalOpen(false);
                  setEditorWhatsappData(null);
                  setEditorWhatsappError(null);
                }}
                className="px-4 py-2 bg-zinc-900 hover:bg-zinc-850 border border-zinc-850 text-zinc-400 hover:text-white text-xs font-mono font-bold rounded-xl transition-all cursor-pointer"
              >
                Close
              </button>
            </div>
          </div>
        </div>
      )}"""

updated = content[:start_idx] + new_content + content[end_idx:]

with open("src/components/ProductionModule.tsx", "w") as f:
    f.write(updated)
print("Updated Footer successfully")
